10/problem10.rb

```diff
@@ -0,0 +1,14 @@
+numbers=[]
+puts "Enter first Number:"
+fnum=Integer(gets.chomp!)
+numbers.push(fnum)
+
+puts "Enter second Number:"
+snum=Integer(gets.chomp!)
+numbers.push(snum)
+
+puts "Enter third Number:"
+tnum=Integer(gets.chomp!)
+numbers.push(tnum)
```

Comment:
You could have use a loop to get (n=3) inputs from user. The benefit of using a loop is, in near or far future, if you want to get 15 numbers from user, you will not have to write 3 * 15 = 45 lines of code.

	1.upto(3) do
	  print 'Please enter number: '
	  numbers << gets.to_i
	end